app/presenters/content_item/contents_list.rb

```diff
@@ -47,14 +47,19 @@ def first_item_has_long_content?
     def first_item_content
       element = first_item
       first_item_text = ""
-      allowed_elements = %w[p ul ol]
-
-      until element.name == "h2"
-        first_item_text += element.text if element.name.in?(allowed_elements)
+      allowed_elements = %w[p ul ol div]
+
+      until element.nil? || element.name == "h2"
+        if element.name == "div"
+          element.children.each do |child|
```

Comment:
It only checks a single level of divs, so it won't find paragraphs inside nested divs. It might solve the immediate problem with the particular document, but the problem will reappear when someone adds a document with more deeply nested divs.

Reply:
Yeah that makes sense - I'll change this so I look for more deeply nested divs as like you said I only considered the first level 😅